src/ci/bin/test.inc.sh

```diff
@@ -112,6 +112,7 @@ cromwell::private::create_build_variables() {
             CROMWELL_BUILD_MYSQL_PASSWORD=""
             CROMWELL_BUILD_MYSQL_SCHEMA="cromwell_test"
             CROMWELL_BUILD_GENERATE_COVERAGE=false
+            CROMWELL_BUILD_SPAN_SCALE_FACTOR="12"
```

Comment:
When I ran the test today, it failed for 1 out of 5 runs. So I have increased the factor from 10->12 (it hasn't failed for 12 yet)

Reply:
Maybe make it 15 or even 20 to be safe?

- I believe these are backstops in case something goes wrong: 
  - In the "normal" case, even making this 10,000 shouldn't slow down the tests (though nb: *shouldn't*) 
  - What making this higher means is that if a test fails because some condition never gets met, we would wait longer before declaring that it will never happen

Reply:
Sounds good. I will increase it to 15.